core/src/test/scala/akka/kafka/internal/ProducerTest.scala

```diff
@@ -61,7 +53,7 @@ class ProducerTest(_system: ActorSystem)
 
   def recordAndMetadata(seed: Int) = {
     new ProducerRecord("test", seed.toString, seed.toString) ->
-      new RecordMetadata(new TopicPartition("test", seed), seed.toLong, seed.toLong, Record.NO_TIMESTAMP, -1, -1, -1)
+      new RecordMetadata(new TopicPartition("test", seed), seed.toLong, seed.toLong, new Date().getTime, java.lang.Long.valueOf(-1), -1, -1)
```

Comment:
Use -1L instead or if it for some weird reason needs to be manually constructed, cache the instance, Long is immutable.

Also, don't allocate one `Date` for every record, use `System.currentTimeMillis` instead.